src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -27,13 +27,15 @@ const partyMapVenueSelector = (state: RootState) =>
   state.firestore.ordered.currentVenue?.[0] as PartyMapVenue;
 
 export const PartyMap: React.FC = () => {
+  const [isRoomModalOpen, setRoomModalOpen] = useState(false);
+  const [selectedRoom, setSelectedRoom] = useState<Room | undefined>();
+
   const { user, profile } = useUser();
+  const { recentVenueUsers } = useRecentVenueUsers();
+
   const currentVenue = useSelector(partyMapVenueSelector);
   const venues = useSelector(orderedVenuesSelector);
 
-  const [isRoomModalOpen, setRoomModalOpen] = useState(false);
-  const [selectedRoom, setSelectedRoom] = useState<Room | undefined>();
-
   const selectRoom = useCallback((room: Room) => {
     setSelectedRoom(room);
     setRoomModalOpen(true);
```

Comment:
Why was the position of the `useState`s moved? They were co-located with their related `useCallback`s below. This should be changed back, unless there is a good reason for it to be moved.